superset/security/manager.py

```diff
@@ -232,7 +233,8 @@ def can_access_all_databases(self) -> bool:
 
     def can_access_database(self, database: "Database") -> bool:
         """
-        Return True if the user can access the Superset database, False otherwise.
+        Return True if the user can access all the tables within the Superset database,
```

Comment:
I though there was merit in being _more_ explicit about what it means to be able to access a database, schema, etc. It's not overly apparent without the comment.